docs/docs/cmd/flow/run/run-get.md

```diff
@@ -97,17 +107,155 @@ m365 flow run get --environmentName Default-d87a7535-dd31-4437-bfe1-95340acd55c5
 === "Text"
 
     ```text
-    endTime    :
-    name       : 08585329112602833828909892130CU17
-    startTime  : 2022-11-17T14:33:45.2763872Z
-    status     : Running
-    triggerName: When_a_new_response_is_submitted
+    endTime    : 2023-03-04T09:05:22.5880202Z
+    id         : /providers/Microsoft.ProcessSimple/environments/Default-e1dd4023-a656-480a-8a0e-c1b1eec51e1d/flows/24335774-daf6-4183-acb7-f5155c2cd2fe/runs/08585236861638480597867166179CU104
+    name       : 08585236861638480597867166179CU104
+    properties : {"startTime":"2023-03-04T09:05:21.8066368Z","endTime":"2023-03-04T09:05:22.5880202Z","status":"Succeeded","correlation":{"clientTrackingId":"08585236861638480598867166179CU131"},"trigger":{"name":"When_an_email_is_flagged_(V4)","inputsLink":{"uri":"https://prod-130.westeurope.logic.azure.com:443/workflows/3ebadb794f6641e0b7f4fda131cdfb0b/runs/08585236861638480597867166179CU104/contents/TriggerInputs?api-version=2016-06-01&se=2023-03-04T15%3A00%3A00.0000000Z&sp=%2Fruns%2F08585236861638480597867166179CU104%2Fcontents%2FTriggerInputs%2Fread&sv=1.0&sig=","contentVersion":"2v/VLXFrKV6JvwSdcN7aHg==","contentSize":343,"contentHash":{"algorithm":"md5","value":"2v/VLXFrKV6JvwSdcN7aHg=="}},"outputsLink":{"uri":"https://prod-130.westeurope.logic.azure.com:443/workflows/3ebadb794f6641e0b7f4fda131cdfb0b/runs/08585236861638480597867166179CU104/contents/TriggerOutputs?api-version=2016-06-01&se=2023-03-04T15%3A00%3A00.0000000Z&sp=%2Fruns%2F08585236861638480597867166179CU104%2Fcontents%2FTriggerOutputs%2Fread&sv=1.0&sig=","contentVersion":"AHZEeWNlQ0bLe48yDmpzrQ==","contentSize":3478,"contentHash":{"algorithm":"md5","value":"AHZEeWNlQ0bLe48yDmpzrQ=="}},"startTime":"2023-03-04T09:05:21.6192576Z","endTime":"2023-03-04T09:05:21.7442626Z","scheduledTime":"2023-03-04T09:05:21.573561Z","originHistoryName":"08585236861638480598867166179CU131","correlation":{"clientTrackingId":"08585236861638480598867166179CU131"},"code":"OK","status":"Succeeded"}}
```

Comment:
@MathijsVerbeeck I think that reverting the removal of `defaultProperties` we should also revert the changes done here as actually this output does not change

```
endTime    : 2023-03-28T22:45:09.5932079Z
name       : 08585215633769253695794073201CU159
startTime  : 2023-03-28T22:45:08.5586405Z
status     : Succeeded
triggerName: manual
```